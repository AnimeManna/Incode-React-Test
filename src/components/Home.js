import React, {Component} from 'react'

class Home extends Component{
    render(){
        return(
            <div>
                Hello Incode Group
            </div>
        )
    }
}

export default Home