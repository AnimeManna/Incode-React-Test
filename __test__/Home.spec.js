import React from 'react'
import Home from '../src/components/Home'
import renderer from 'react-test-renderer';
import toJson from 'enzyme-to-json';
import {shallow} from 'enzyme'

describe('<Home />', ()=>{
    describe('render()', ()=>{
        test('render the components',()=>{})
        const wrapper = shallow(<Home />)
        const component = wrapper.dive()

        expect(toJson(component)).toMathSnapshot();
    })
})



