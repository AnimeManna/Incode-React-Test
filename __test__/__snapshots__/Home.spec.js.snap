// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> render() render the components 1`] = `
<div
  className="ui center aligned segment"
>
  Welcome Incode-Group sure you will like it)
</div>
`;
